packages/runtime/test-runtime-utils/src/mocksDataStoreContext.ts

```diff
@@ -96,7 +96,9 @@ export class MockFluidDataStoreContext implements IFluidDataStoreContext {
 		throw new Error("Method not implemented.");
 	}
 
-	// back-compat: to be removed in 2.0
+	/**
+	 * @deprecated back-compat: to be removed in 2.0
```

Comment:
Not removed in 2.0. I think you can list 2.20. There should be an issue added to 2.20 removals.